subgraphs/tokemak/subgraph.yaml

```diff
@@ -0,0 +1,60 @@
+specVersion: 0.0.2
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: Manager
+    network: mainnet
+    source:
+      address: "0xA86e412109f77c45a3BC1c5870b880492Fb86A14"
+      startBlock: 12912471
+      abi: Manager
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      entities:
+        - NewVault
+      abis:
+        - name: Manager
+          file: ./abis/Manager.json
+        - name: Vault
+          file: ./abis/Vault.json
+        - name: ERC20
+          file: ./abis/ERC20Detailed.json
+      eventHandlers:
+        - event: PoolRegistered(address)
+          handler: handleNewVault
```

Comment:
this should be called `handlePoolRegistered`

Reply:
Done